src/overlays/actors/ovl_Bg_Jya_Goroiwa/z_bg_jya_goroiwa.c

```diff
@@ -21,25 +33,181 @@ const ActorInit Bg_Jya_Goroiwa_InitVars = {
     (ActorFunc)BgJyaGoroiwa_Update,
     (ActorFunc)BgJyaGoroiwa_Draw,
 };
-*/
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_Bg_Jya_Goroiwa/func_80897970.s")
+extern Gfx D_060006B0[];
+
+static ColliderJntSphItemInit sJntSphItemsInit[] = {
+    {
+        { 0x00, { 0x20000000, 0x00, 0x04 }, { 0x00000000, 0x00, 0x00 }, 0x01, 0x00, 0x01 },
+        { 0, { { 0, 0, 0 }, 58 }, 100 },
+    },
+};
+
+static ColliderJntSphInit sJntSphInit = {
+    { COLTYPE_UNK10, 0x11, 0x00, 0x39, 0x20, COLSHAPE_JNTSPH },
+    1,
+    sJntSphItemsInit,
+};
+
+static CollisionCheckInfoInit collisionCheckInfoInit = { 0x01, 0xF, 0x0, 0xFE };
+
+static InitChainEntry initChain[] = {
```

Comment:
We generally call CollisionCheckInfoInit variables `sColChkInfoInit`.
Also, in our naming scheme, all "static" variables (in the sense of local to the file) should start with `s`, while global variables start with `g`
```suggestion
static CollisionCheckInfoInit sColChkInfoInit = { 0x01, 0xF, 0x0, 0xFE };

static InitChainEntry sInitChain[] = {
```